core/domain/email_manager.py

```diff
@@ -1732,6 +1732,22 @@ def send_account_deleted_email(user_id, user_email):
         recipient_email=user_email)
 
 
+def send_account_deletion_failed_email(user_id, user_email):
+    """Sends an email to admin about the failure of deletion of the user.
+
+    Args:
+        user_id: str. The id of the user whose account failed to get deleted.
+        user_email: str. The email of the user whose account failed to
+            get deleted.
+    """
+    email_subject = 'WIPEOUT: Account deletion failed'
+    email_body_template = (
+        'The Wipeout of the user with ID \'%s\' and email \'%s\' failed.'
```

Comment:
Nit: The Wipeout process failed for the user with ID \'%s\' and email \'%s\'.

Reply:
Done.